lib/brasil_api/taxas.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+module BrasilAPI
+  class Taxas < Base
+    class << self
+      def taxs(name=nil)
```

Comment:
No caso como aqui referencia taxas de juros, em inglês seria `rates`, taxs faz referência a impostos.
```suggestion
      def rates(name=nil)
```